packages/datadog-instrumentations/src/orchestrion-config/index.js

```diff
@@ -0,0 +1,5 @@
+const path = require('path')
+const fs = require('fs')
+
+// TODO this needs to be inlined to prevent issues in bundling
```

Comment:
Doesn't the native addon itself have the same issue?

Reply:
The rewriter is entirely wasm, not an addon, but yeah, maybe. This comment is more a reminder to think about this sort of thing, and not necessarily a future requirement. Would you like me to reword it?